### PR TITLE
Adds some basic medicine autoinjectors/bottles to the code so I can deviously try to add them to ops orders later.

### PR DESCRIPTION
**Pull request by anconfuzedrock**
Based on feedback from
https://forums.aurorastation.org/topic/17493-allow-operations-to-order-more-complicated-medicines
I have finally gotten off my ass and added some bottles and injectors, so that I can later attempt to have these to operations orders as **expensive** emergency items. These are:
-15u injectors of bicaridine and kelotane
-10 injectors of peridaxon
-60u bottles of hyronalin
the injectors coming in injector form so they're less versatile and can't feasibly subvert having a pharmacist. This pr itself doesn't add them to ops yet, though that's basically the point. I'm thinking like 1000 credits for a peridax injector, 500 for one of the other injectors or the hyronalin bottle?

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by anconfuzedrock** [comment]
!review

**Comment by anconfuzedrock** [comment]
worth a shot

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -321,4 +321,29 @@
...
-	reagents_to_add = list(/decl/reagent/sanasomnum = 20)
\ No newline at end of file
+	reagents_to_add = list(/decl/reagent/sanasomnum = 20)
+
+/obj/item/reagent_containers/hypospray/autoinjector/bicaridine
+	name = "bicaridine autoinjector"
+	desc = "An autoinjector loaded with bicaridine, a chemical used to treat physical trauma."
+	volume = 15
+	amount_per_transfer_from_this = 15
+
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/hypospray.dm:331]:
```suggestion
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -321,4 +321,29 @@
...
+	reagents_to_add = list(/decl/reagent/bicaridine = 15)
+
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/hypospray.dm:333]:
```suggestion
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -321,4 +321,29 @@
...
+
+/obj/item/reagent_containers/hypospray/autoinjector/kelotane
+	name = "kelotane autoinjector"
+	desc = "An autoinjector loaded with kelotane, a chemical used to treat burnt tissue."
+	volume = 15
+	amount_per_transfer_from_this = 15
+
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/hypospray.dm:340]:
```suggestion
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -321,4 +321,29 @@
...
+	reagents_to_add = list(/decl/reagent/kelotane = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/peridaxon
+	name = "peridaxon autoinjector"
+	desc = "An autoinjector loaded with peridaxon, a chemical used to treat minor organ damage."
+	volume = 10
+	amount_per_transfer_from_this = 10
+
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/hypospray.dm:348]:
```suggestion
```

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (anconfuzedrock) [reply code/modules/reagents/reagent_containers/hypospray.dm:332]: yea my git's a little bit fucked right now after I had to wipe my pc so ima just try this commit suggestion button

**Comment by anconfuzedrock** [comment]
on account of my git being highly fucked, I milked that commit suggestion button.

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**